utils/io.py

```diff
@@ -47,7 +49,7 @@ def load_checkpoint(model, checkpoint_path, use_cuda=False):
     return model, state
 
 
-def save_model(model, optimizer, current_step, epoch, r, output_path, **kwargs):
+def save_model(model, optimizer, current_step, epoch, r, output_path, amp_state_dict=None, **kwargs):
```

Comment:
you don't need to pass amp_state_dict. It is better to use **kwargs so we can get rid of the default value.

Reply:
ok, uses kwargs now